classes/pdf/PDFGenerator.php

```diff
@@ -147,7 +148,7 @@ public function setFontForLang($iso_lang)
         if (array_key_exists($iso_lang, $this->font_by_lang)) {
             $this->font = $this->font_by_lang[$iso_lang];
         }else {
-            $this->font = self::DEFAULT_FONT;
+            $this->font = 'dejavusans';
```

Comment:
@Quetzacoalt91 I didn't want to change the DEFAULT_FONT, all I want is to prevent that error from occur and I didn't want to add every language with its font, so if font_by_lang is not definied then we use 'dejavusans' as font for PDF ;)